staging/src/k8s.io/apimachinery/pkg/runtime/types.go

```diff
@@ -102,6 +102,7 @@ type RawExtension struct {
 	// Raw is the underlying serialization of this object.
 	//
 	// TODO: Determine how to detect ContentType and ContentEncoding of 'Raw' data.
+	// +listType=atomic
```

Comment:
these look ~fine for now, but we could also fix the linter to not complain about `[]byte` "lists". cc @apelisse

Reply:
https://github.com/kubernetes/kube-openapi/pull/437

Reply:
I will back these out from this PR, but as a new commit for readability